embrace-gradle-plugin/src/main/java/io/embrace/android/gradle/plugin/instrumentation/config/ConfigClassVisitorFactory.kt

```diff
@@ -17,12 +18,14 @@ object ConfigClassVisitorFactory {
         NetworkCaptureConfig,
         ProjectConfig,
         RedactionConfig,
-        SessionConfig;
+        SessionConfig,
+        Base64SharedObjectFilesMap;
 
         val className = "io.embrace.android.embracesdk.internal.config.instrumented.${name}Impl"
 
         fun createClassVisitor(
             cfg: VariantConfig,
+            encodedSharedObjectFilesMap: String?,
```

Comment:
This should be added onto `VariantConfig` like other values set via the gradle plugin and not embrace-config.json - e.g. `buildId`

Reply:
I agree it would be better to have it in VariantConfig, but it might be hard to achieve that.

The encoding task depends on VariantConfig, so we can't wait for the output of the task before creating VariantConfig. I suppose we could add a field that it's initialized later, but it might end up being too complex.

I'd leave it like it is for now.

Reply:
Happy to go with this for now in that case - we can address this in future